packages/fiber/src/core/index.tsx

```diff
@@ -308,10 +308,18 @@ function createRoot<TCanvas extends Canvas>(canvas: TCanvas): ReconcilerRoot<TCa
         if ('enabled' in ColorManagement) ColorManagement.enabled = !legacy
         else if ('legacyMode' in ColorManagement) ColorManagement.legacyMode = legacy
       }
-      const outputEncoding = linear ? THREE.LinearEncoding : THREE.sRGBEncoding
-      const toneMapping = flat ? THREE.NoToneMapping : THREE.ACESFilmicToneMapping
-      if (gl.outputEncoding !== outputEncoding) gl.outputEncoding = outputEncoding
-      if (gl.toneMapping !== toneMapping) gl.toneMapping = toneMapping
+
+      if ('outputColorSpace' in gl) {
+        const LinearSRGBColorSpace = 'srgb-linear'
+        const SRGBColorSpace = 'srgb'
+        gl.outputColorSpace = linear ? LinearSRGBColorSpace : SRGBColorSpace
+      } else {
+        const LinearEncoding = 3000
+        const sRGBEncoding = 3001
+        gl.outputEncoding = linear ? LinearEncoding : sRGBEncoding
```

Comment:
Need to see whether we should hard code encoding constants or if they'll remain following `Texture.encoding` => `Texture.colorSpace`.